FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2TextFieldActivity.kt

```diff
@@ -303,7 +303,9 @@ class V2TextFieldActivity : V2DemoActivity() {
                             ),
                             keyboardOptions = KeyboardOptions(keyboardType = keyboardType),
                             keyboardActions = KeyboardActions(onAny = { focusManager.clearFocus() }),
-                            visualTransformation = if (passwordMode) PasswordVisualTransformation() else VisualTransformation.None
+                            visualTransformation = if (passwordMode) PasswordVisualTransformation() else VisualTransformation.None,
+                            readOnly = false,
+                            enabled = true,
```

Comment:
You can add these values in the modifiable parameter section. Also add test cases

Reply:
Done